### PR TITLE
Adding --chain option

### PR DESCRIPTION
**Pull request by ddorgan**

### CODE REVIEW

**Comment by keorn** [comment]
Cant it be more generic and just pass all options to the underlying nodes?

**Comment by ddorgan** [comment]
Ok to use --options or something like that? It's a bit of a pain to pass everything down due to how the arguments are being parsed...

**Comment by keorn** [comment]
Sure

File: README.md [hunk 1]
```diff
@@ -23,13 +23,21 @@ Required:
...
 ```--name``` This option allows you to set the name of the chain in use. Default chain name is parity.
 
 ```--nodes``` The amount of nodes that will be used with an aura or tendermint network. By default one non-authority node is also created. Default value is 2 nodes.
 
-```--ethstats``` This option will start an ethstats monitoring instance that can be connected to on http://localhost:3001 
+```--ethstats``` This option will start an ethstats monitoring instance that can be connected to on http://localhost:3001
+
+```--expose``` This option will expose ports 8080, 8180, 8545 and 30303 on the container name given as an argument.
+
+``--chain-options``` Pass a quoted string of options for the parity container, e.g. "--force-signing --no-ui"
```
Comment (keorn) [inline README.md]: Why not `--parity-options`? e.g.`--no-ui` is not a chain option.

File: README.md [hunk 2]
```diff
@@ -23,13 +23,21 @@ Required:
 * tendermint - Tendermint is another consensus engine that can be used with parity, however not as well tested as Authority Round.  
 * validatorset - Validtor Set is used to transition a chain from one validator to another at a certain block.
 * input.json - Use a custom input chain spec file used on all of the nodes.
+OR
+```---chain``` This option allows you to use a chain built into parity, e.g. foundation, kovan, classic
```
Comment (keorn) [inline README.md]: This can be passed via `--parity-options`, no need to put it here.

File: parity-deploy.sh [hunk 3]
```diff
@@ -245,16 +251,22 @@ while [ "$1" != "" ]; do
         --expose)               shift
                                 EXPOSE_CLIENT="$1"
                                 ;;
-        -h | --help )           help 
+				--chain)                shift
```
Comment (keorn) [inline parity-deploy.sh]: Is it still needed? Doesn't it go to `--parity-options`?

**Comment by ddorgan** [comment]
Well I thought the idea was you could use just use --chain chain_name for a simple default and only use --parity-options if you want something different from the default.

Or it can be removed and people will need to use --parity-options "--chain chain_name --other-options" 

Any preference?

**Comment by keorn** [comment]
Cant you just pass all unknown options to Parity?

**Comment by ddorgan** [comment]
Well the one advantage of keeping the --chain out of --parity-options is that it can then be used with custom networks too to enable a certain option on all the containers (e.g. debug logging), without needing to touch the template files.

**Pull request merged**